components/textfield/stories/template.js

```diff
@@ -140,6 +156,15 @@ export const Template = ({
 				})}
 			/>`
 		)}
+		${when(helpText, () => HelpText({
+			...globals,
```

Comment:
I was going to say, I don't think we need the `...globals`, but that's just on main now, right?! 😂🤦‍♀️